infra/global-metrics/data/grafana/grafana.ini

```diff
@@ -7,6 +7,6 @@ default_home_dashboard_path = /etc/grafana/dashboards/ursa.json
 # allow anonymous users to view dashboards
 [auth.anonymous]
 enabled = true
-org_name = Main Org.
+org_name = Fleek Network.
```

Comment:
need to setup this org as the main org as well separately

Reply:
can we rename this or is this a hard requirement

Reply:
Hard requirement to have it the default `Main Org.` or you'd need to add an overall definition for the group if changing. It's just a semantics thing tho so reverting should be fine, not too important

Reply:
[49dcb9c](https://github.com/fleek-network/ursa/pull/217/commits/49dcb9c51a65334bd80e5d491bad20cf14fad7e0)